src/components/SRNEntry.vue

```diff
@@ -54,7 +54,7 @@
     }}</span>
     <span
       class="mx-auto text-red-700 text-base mb-1"
-      v-if="!isUserValid && validateCount == 1"
+      v-if="!isCurrentUserValid && validateCount == 1"
```

Comment:
might be better (in terms of readability) to wrap this within a computed property with a name (like): `isInvalidLoginMessageShown`